index.html

```diff
@@ -179,6 +196,67 @@ <h3>
           }
         </pre>
       </section>
+      <section>
+        <h3>
+          Processing the `note_taking` member
+        </h3>
+        <p>
+          To <dfn>process the `note_taking` member</dfn>, given [=ordered map=]
+          |json:ordered map|, [=ordered map=] |manifest:ordered map|, [=URL=]
+          |manifest_URL:URL|, run the following during the
+          <a data-cite=
+          "appmanifest#dfn-processing-extension-point-of-web-manifest">extension
+          point</a> in [=processing a manifest=]:
+        </p>
+        <ol class="algorithm">
+          <li>If |json|["note_taking"] does not [=map/exist=], return.</li>
+          <li>If the type of |json|["note_taking"] is not [=ordered map=],
+            return.
+          </li>
+          <li>Set |manifest|["note_taking"] to a new [=ordered map=].</li>
+          <li>If |json|["note_taking"]["new_note_url"] does not [=map/exist=],
```

Comment:
I think this should be put into a function ("processing the new note URL"). Because if we ever add a second member to `note_taking` that's independent of `new_note_url`, this will break since you have all these return statements on failure.

Instead, move this into a function, then all of those return statements will just return out of "processing the new note URL" rather than the whole processing function.

Reply:
Done.